src/lib/wallet.ts

```diff
@@ -0,0 +1,34 @@
+import chalk from 'chalk';
+import qrcode from 'qrcode-terminal';
+import { Wallet, ethers } from 'ethers';
+import { getPrivateKey } from './keys.js';
+import { networks } from './constants.js';
+
+export const getWallet = async (): Promise<Wallet | null> => {
+  const privateKey = await getPrivateKey();
+  if (!privateKey) return null;
+
+  return new Wallet(privateKey);
+};
+
+export const displayWallet = async (): Promise<void> => {
+  const wallet = await getWallet();
+  const address = wallet?.address ?? 'No wallet found';
+  console.log('Wallet address:', chalk.green(address));
+  console.log();
```

Comment:
Why this empty console logs?

Reply:
+1

Reply:
was to give an extra line, adding escaped char in the previous